bucket/gallery-dl.json

```
@@ -6,6 +6,10 @@
     "url": "https://github.com/mikf/gallery-dl/releases/download/v1.26.7/gallery-dl.exe",
     "hash": "d4bb6b7097d03cd305b07e07d64938dc51ac02d52b7521538b9ceedbd3e8f583",
     "bin": "gallery-dl.exe",
+    "pre_install": "if (-not (Test-Path \"$persist_dir\\gallery-dl.conf\")) { New-Item \"$dir\\gallery-dl.conf\" -ItemType file | Out-Null }",
```

```suggestion
    "pre_install": "if (-not (Test-Path \"$persist_dir\\gallery-dl.conf\")) { Set-Content \"$dir\\gallery-dl.conf\" '{}' -Encoding Ascii | Out-Null }",
```
Otherwise we get `[config][error] JSONDecodeError when loading 'C:\Users\$USER\scoop\apps\gallery-dl\current\gallery-dl.conf': Expecting value: line 1 column 1 (char 0)` because the file is empty